docs/tutorials/fundamentals/lifecycle-methods.md

```diff
@@ -134,8 +136,9 @@ Once installed, import the package, preferably within your startup script:
 === "Luau"
 
     ```Lua
+    local ReplicatedStorage = game:GetService("ReplicatedStorage")
     -- this assumes you've aliased the package as `lifecycles`
-    require(ReplicatedStorage.Packages.lifecycles)
+    local lifecycles = require(ReplicatedStorage.Packages.lifecycles)
```

Comment:
the docs dont give a reference for lifecycles as you arent meant to use it, instead youre meant to just import for its side effects